packages/rfw/test_coverage/bin/test_coverage.dart

```diff
@@ -9,7 +9,7 @@ import 'package:lcov_parser/lcov_parser.dart' as lcov;
 // Please update these targets when you update this package.
 // Please ensure that test coverage continues to be 100%.
 const int targetLines = 2127;
-const String targetPercent = '100';
+const String targetPercent = '64';
```

Comment:
You should file a bug explaining the context and reference it in a TODO. Generally anything that's intended to be temporary should be tracked.

Reply:
The `TODO(@Hixie): Update tests to return coverage to 100%` in the issue should be here as a comment on this line (with a link to the issue).